docs/meta/schema_style_guide.rst

```diff
@@ -296,16 +296,16 @@ For the ``description`` field of an object:
 
 .. code-block:: none
 
-   A description of this <object_name>. Structured information should be provided in <related_fields>.
+   A description of this <object_name>.
 
 **Examples:**
 
-   A description of this tender. Structured information should be provided in the items array. Descriptions should be short and easy to read. Avoid using ALL CAPS.
+   A summary description of the tender. This complements any structured information provided using the items array. Descriptions should be short and easy to read. Avoid using ALL CAPS.
```

Comment:
The second sentence is not really a desirable example to follow. We can either remove this example or find another one.

Reply:
Given that the template is now simply 'A description of this <object_name>', I'm not sure either example is needed. What do you think?

Reply:
I think the examples demonstrate some of the customization that can be made, though I think in most cases we will just use the template. I'm fine to keep them.

Reply:
Okeydokey. Assuming that you still want to replace this one with a more desirable example, how about `Bid.description`:

> A description of this bid. The description field may be used to describe the terms and conditions for the bid.

Reply:
Hmm, I'd probably rewrite it as:

"A description of the bid, which may include terms and conditions."

Though to be honest I don't know what is meant by terms and conditions in the context of a bid.

Maybe just leave the one document example.